### ISSUE TITLE
Implement new page for downloading presidential public fund submission files in Wagtail

### ISSUE THREAD

**Issue by jenniferthibault** [fecgov/fec-cms#1406]
Moved from design and content modeling issues in https://github.com/18F/openFEC-web-app/issues/2318
______
### Designs
🎨 [**InVision Prototype**](https://gsa.invisionapp.com/share/K5DYFESA8#/258356505_Pres-Matching-Fund-Sub) of Presidential matching fund submissions page

### Page sources on classic
- [2016](http://classic.fec.gov/finance/disclosure/2016MatchingFundSubmissions.shtml)
- [2012](http://classic.fec.gov/finance/2012matching/2012matching.shtml)
- [2008](http://classic.fec.gov/finance/2008matching/2008matching.shtml)
- [2004](http://classic.fec.gov/finance/disclosure/2004MatchingFundSubmissions.shtml)
- [2000](http://classic.fec.gov/finance/disclosure/submiss.shtml)
_______

### Completion criteria

**Migrate the content and downloads to a new site**
- [x] In Wagtail, make a child page of the [Presidential public funding page](https://www.fec.gov/press/resources-journalists/presidential-public-funding/) using the "Custom page" template

  - Page title: `Presidential matching fund submissions`
  - Intro copy: `In addition to regular quarterly or monthly disclosure reports, presidential candidates who seek matching funds must submit information about matchable contributions to the FEC for review. `
  - Link: `Learn about the presidential public funding  program  »` back to https://www.fec.gov/press/resources-journalists/presidential-public-funding/

- [x] Use custom HTML tables to format presidential public funds submissions back to 2000. Match to designs in prototype linked above.
- [x] Link to downloads that live in the new s3 environments (not Waltham)
- [ ] Publish the new page!

**Then, help people get to the new page**
- [ ] In Wagtail, add Related topic link and in copy link to new page on [Presidential public funding page
](https://www.fec.gov/press/resources-journalists/presidential-public-funding/)
  - ![screen shot 2017-10-26 at 11 12 58 pm](https://user-images.githubusercontent.com/11636908/32086467-669f544a-baa3-11e7-895f-cad3e8a0f279.png)
- [x] In HTML files, add link to new page from Candidates tab of Advanced data page
  - ![screen shot 2017-10-26 at 11 14 41 pm](https://user-images.githubusercontent.com/11636908/32086490-9a4ed220-baa3-11e7-9480-4ead50eeb661.png)

**Comment by jwchumley** [comment]
Dependency: Fixing the HTML wagtail bug.

**Comment by johnnyporkchops** [comment]
Draft page: 
https://fec-prod-proxy.app.cloud.gov/admin/pages/9369/view_draft/

**Comment by jenniferthibault** [comment]
@johnnyporkchops you're very close on the sidebar formatting. I inspected another page using the same module successfully, and found that the secret to getting it to render correctly is to use the `Heading` block separately from the body.

**Correct formatting use**

<img width="1059" alt="screen shot 2017-11-13 at 3 51 20 pm" src="https://user-images.githubusercontent.com/11636908/32748911-b18d4430-c88b-11e7-8802-438340f2d97a.png">


**Current formatting use**
<img width="1055" alt="screen shot 2017-11-13 at 3 58 29 pm" src="https://user-images.githubusercontent.com/11636908/32749004-ecc64ce0-c88b-11e7-9459-01b94e017dc3.png">

**Comment by jenniferthibault** [comment]
Also worth noting that this issue will be blocked from 100% completion until we can link to this page from the Candidates section of the Advanced data page. (The last completion criteria above)

That was intended to require the restyling in https://github.com/18f/fec-cms/issues/1309 ... though it's possible to add it in using the current styles.

**Comment by llienfec** [comment]
Talked to the auditor that handles the Title 26 audits.  Her feedback was incredibly positive.  Said she loved how clear and organized these pages were, everything makes sense, everything is in one place and you feel like you found everything you need.  She said she's really impressed with the new site and thinks it's awesome for users.  Really good job all!

cc: @AmyKort

**Comment by johnnyporkchops** [comment]
The wagtail drafts of the two pres funding pages are here:
New pres public funding submission page: 
https://fec-prod-proxy.app.cloud.gov/admin/pages/9369/view_draft/
Draft of pres public funding page w/ new link:
https://fec-prod-proxy.app.cloud.gov/admin/pages/113/revisions/15921/view/

Here is a screenshot for the candidates tab above in the [adv data page PR ](https://github.com/18F/fec-cms/pull/1603)
![candidates_tab](https://user-images.githubusercontent.com/5572856/33976330-240a7134-e062-11e7-93fa-e1e9598fcfc8.gif)

**Comment by PaulClark2** [comment]
What's the status of this issue?

**Comment by AmyKort** [comment]
What's the status of this issue?

**Comment by llienfec** [comment]
My part of getting the approvals from Audit is finished!

**Comment by AmyKort** [comment]
Is publication of this page blocked by work to complete the advanced data pages?

**Comment by jenniferthibault** [comment]
@johnnyporkchops can you speak to this? 
It looks like from your screenshot above that the link is implemented in the Adv data page page, and the link is live from the Press page here: https://www.fec.gov/press/resources-journalists/presidential-public-funding/

**Comment by AmyKort** [comment]
The link from the press page is returning a 404. Will that be resolved when we publish the Advanced Data pages?

![image](https://user-images.githubusercontent.com/13237823/38310506-be2e4ff8-37ea-11e8-989b-18f99e119dd6.png)

**Comment by llienfec** [comment]
Oh!  I saw a section for this when I was looking at bulk downloads this morning.  What would you think of sending it here:
![image](https://user-images.githubusercontent.com/22914750/38311279-cc32cc3a-37ec-11e8-97d6-3d3c486bd521.png)

The only downside is that it's lower on this page: https://www.fec.gov/data/advanced/?tab=other
We're not able to send someone directly to the presidential section of the page because anchor tags don't work in Wagtail.  Still worth the link?

**Comment by AmyKort** [comment]
What's the status of this issue?

**Comment by llienfec** [comment]
The broken link above looks like it doesn't exist anymore with the completion of the presidential public funding section.

@johnnyporkchops: can you speak to the rest of this issue?

**Comment by dorothyyeager** [comment]
I actually just noticed that link as well.  Press has discontinued that page since we've got the public funding section up. I'd recommend linking it to https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/#reports-and-resources (since that will take them to the part where they can access list of public funding recipients).
